### PR TITLE
Release note

### PR DESCRIPTION
**Pull request by sepandhaghighi**
#### What does this implement/fix? Explain your changes.
- `RELEASE.md` added

### CODE REVIEW

File: otherfile/RELEASE.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
+
```
Comment (sadrasabouri) [inline otherfile/RELEASE.md:1]: Remove this extra line.

File: otherfile/RELEASE.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+	7. Squash and merge
+	8. Delete `release` branch
+5. Merge `dev` branch into `master`
+	1. Checkout to `master`
+	2. `git merge dev`
+	3. `git push origin master`
+	4. Wait for all CI pass
+6. Build EXE file
+	1. Use `Python 3.4.x`
+	2. Run `build_exe.bat`
```
Comment (sadrasabouri) [inline otherfile/RELEASE.md:32]: Change it to
```
1. Run `build_exe.bat` (Use `Python 3.4.x`)
```

File: otherfile/RELEASE.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+	4. Copy changelogs
+	5. Tag all related issues
+	6. Upload EXE file
+8. Bump!!
+9. Close this version issues
+10. Close milestone
+11. Update website
+	1. `git checkout gh-pages`
+	2. Update all version tags
+	3. Update size of files
```
Comment (sadrasabouri) [inline otherfile/RELEASE.md:46]: Be more specific here. Where is version tags and file sizes.

**Review by sadrasabouri — CHANGES_REQUESTED** [review]
Thanks for your efforts.
Minor changes are needed.

File: otherfile/RELEASE.md (hunk 2 above)
Comment (sepandhaghighi) [reply otherfile/RELEASE.md]: fixed in https://github.com/sepandhaghighi/pyrgg/pull/139/commits/3325f4e7647bef2be9d6832bf7e45ac3df4570a9

File: otherfile/RELEASE.md (hunk 3 above)
Comment (sepandhaghighi) [reply otherfile/RELEASE.md]: fixed in https://github.com/sepandhaghighi/pyrgg/pull/139/commits/3325f4e7647bef2be9d6832bf7e45ac3df4570a9

**Pull request merged**